### PR TITLE
[10.0] Migrate mrp_bom_note

### PR DESCRIPTION
**Pull request by bimajatiwijaya**

### CODE REVIEW

File: mrp_bom_note/README.rst [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
 ==========================
 
 This module creates in BoM a new field to add notes
 
+Installation
+============
+
+To install this module, you need to:
+
+#. install mrp application
```
Comment (elicoidal) [inline mrp_bom_note/README.rst]: Here: not really need for enumerated list (there is only one element) ...

File: mrp_bom_note/README.rst [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
+=============
+
+No need configure required.
+
+
 Usage
 =====
 
-* Go to Manufacturing > Products > Bill of Materials
+* Go to Manufacturing > Master Data > Bill of Materials
```
Comment (elicoidal) [inline mrp_bom_note/README.rst]: ... but here it would be convenient actually to have one (enumerated list)

File: mrp_bom_note/README.rst [hunk 3]
```diff
@@ -1,22 +1,41 @@
...
 * Edit an existing BoM and fill in the Notes field on tab Notes
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
    :target: https://runbot.odoo-community.org/runbot/129/9.0
 
+Known issues / Roadmap
```
Comment (elicoidal) [inline mrp_bom_note/README.rst]: if empty please delete the section

File: mrp_bom_note/README.rst [hunk 4]
```diff
@@ -28,12 +47,18 @@ If you spotted it first, help us smashing it by providing a detailed and welcome
...
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
 Contributors
 ------------
 
 * Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
 * Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
 * Ana Juaristi <anajuaristi@avanzosc.es>
+* Bima Wijaya <bimajatiwijaya@gmail.com>
```
Comment (elicoidal) [inline mrp_bom_note/README.rst:55]: below please update http=>https

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details. LGTM

**Comment by bimajatiwijaya** [comment]
Thanks for your review @elicoidal . i update it.

File: mrp_bom_note/README.rst [hunk 5]
```diff
@@ -1,17 +1,28 @@
...
 
+Installation
+============
+
+To install this module, you need to install mrp application.
+
+Configuration
+=============
+
+No need configure required.
```
Comment (elicoidal) [inline mrp_bom_note/README.rst]: No configuration required

File: mrp_bom_note/README.rst [hunk 6]
```diff
@@ -1,17 +1,28 @@
...
+
+
 Usage
 =====
 
-* Go to Manufacturing > Products > Bill of Materials
-* Edit an existing BoM and fill in the Notes field on tab Notes
+1. Go to Manufacturing > Master Data > Bill of Materials then edit an existing BoM and fill in the Notes field on tab Notes
```
Comment (elicoidal) [inline mrp_bom_note/README.rst]: you misunderstood my point:
```
#. Go to Manufacturing > Master Data > Bill of Materials then 
#. Edit an existing BoM and 
#. Fill in the Notes field on tab Notes

**Review by elicoidal — CHANGES_REQUESTED** [review]

File: mrp_bom_note/__manifest__.py [hunk 7]
```diff
@@ -14,6 +14,7 @@
         "Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>",
         "Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>",
         "Ana Juaristi <anajuaristi@avanzosc.es>",
+        "Bima Wijaya <bimajatiwijaya@gmail.com>",
```
Comment (andhit-r) [inline mrp_bom_note/__manifest__.py]: You should remove contributor key. Contributor list already mention on README

**Review by andhit-r — CHANGES_REQUESTED** [review]

**Review by andhit-r — APPROVED** [review]
LGTM

**Comment by LoisRForgeFlow** [comment]
@elicoidal Could you update your review?

**Review by elicoidal — APPROVED** [review]
LGTM

**Review by LoisRForgeFlow — APPROVED** [review]

**Comment by LoisRForgeFlow** [comment]
commits squashed on merge

**Pull request merged**